core/src/main/java/org/mobilitydata/gtfsvalidator/validator/DefaultTableHeaderValidator.java

```diff
@@ -34,16 +34,16 @@ public class DefaultTableHeaderValidator implements TableHeaderValidator {
   public void validate(
       String filename,
       CsvHeader actualHeader,
-      Set<String> supportedColumns,
-      Set<String> requiredColumns,
+      Set<String> supportedHeaders,
+      Set<String> requiredHeaders,
```

Comment:
Does it make sense to change the naming from "columns" to "headers" when the rest of the method still uses "column" as its naming convention?

Reply:
I reverted the change.  I think Header *may* be more accurate as the class is validating headers, but since the notice uses the term "column" and the terms are somewhat mixed and matched, I figured I'd go for consistency in this class.